src/subscription.ts

```diff
@@ -528,27 +580,33 @@ export class Subscription extends Subscriber {
    *   const apiResponse = data[0];
    * });
    */
-  getMetadata(gaxOpts?, callback?) {
+  getMetadata(callback?: RequestCallback<Subscription>):
+    void | Promise<Subscription>;
+  getMetadata(gaxOpts: CallOptions, callback: RequestCallback<Subscription>):
+    void;
+  getMetadata(
```

Comment:
Do you wanna keep consistent here?
* `callback `=> void
* `gaxOpt?` => promise
* `gaxOpts`, `callback `=> void
* [gaxOptsOrCallback?](url), `callback?` => void|Promise +> implementation